e2e/main_test.go

```diff
@@ -22,12 +21,14 @@ func TestMain(m *testing.M) {
 
 	flag.Parse()
 
-	logger, err := zap.NewDevelopment()
+	log, err := test.CreateLogger()
 	if err != nil {
 		panic(err)
 	}
-
-	log := logger.Sugar()
+	
+	defer func() {
+		_ = log.Sync()
+	}()
```

Comment:
Interesting that the linter does not complain about now, but when we discussed it you had a linter error. Since the linter is happy we can keep this.